Love_Sonnet_XI_by_Pablo_Neruda

```diff
@@ -0,0 +1,19 @@
+Love Sonnet XI by Pablo Neruda
+
```

Comment:
Could you add a separator between the title and the poem? Maybe dashes?

Reply:
I added some dashes. How did I do?

Reply:
Fantastic!